src/ui.js

```diff
@@ -502,6 +502,18 @@ UIMode.Command = class extends UIMode {
     }
 };
 
+// We are viewing a diagram embedded in another page.
+UIMode.Embed = class extends UIMode {
```

Comment:
```suggestion
UIMode.Embedded = class extends UIMode {
```
I think this is slightly more descriptive.